resources/testdriver.js

```diff
@@ -657,31 +657,31 @@
          */
         in_automation: false,
 
-        click: function(element, coords) {
+        async click(element, coords) {
             if (this.in_automation) {
-                return Promise.reject(new Error('Not implemented'));
+                throw new Error("click() can't be called in automation");
             }
 
             return new Promise(function(resolve, reject) {
                 element.addEventListener("click", resolve);
             });
         },
 
-        delete_all_cookies: function(context=null) {
-            return Promise.reject(new Error("unimplemented"));
+        async delete_all_cookies(context=null) {
+            throw new Error("delete_all_cookies() is not implemented by test_driver_internal");
```

Comment:
We should perhaps make it more explicit that the real problem is it isn't implemented by `testdriver-vendor.js` (or whatever the file is called), rather than mentioning the (most irrelevant) existence of the `test_driver_internal` object?

Reply:
That's a good point.